src/flint/fmpz_mod.jl

```diff
@@ -342,6 +342,20 @@ function zero!(z::ZZModRingElem)
   return z
 end
 
+function one!(z::ZZModRingElem)
+  one!(z.data)
+  return z
+end
+
+function neg!(z::ZZModRingElem, a::ZZModRingElem)
+  if iszero(a.data)
+    z.data = zero!(z.data)
+  else
+    z.data = sub!(z.data, R.n, a.data)
+  end
+  return z
```

Comment:
no idea if this is correct (seems non-trivial enough at least to deserve a comment)